client/src/components/Sidebar/modules/Search.tsx

```diff
@@ -45,6 +46,11 @@ export const SearchInput: React.FunctionComponent<SearchInputProps> = ({
       action={searchUrl}
       method="get"
       onSubmit={onSubmitForm}
+      data-controller="w-kbd"
+      aria-keyshortcuts="/"
```

Comment:
Let's move this aria one above to under `action` so we keep the data attributes grouped together.

Let's also add a comment as to why we are doing this `action-value` dynamic behaviour.

These are small items, I can do when merging.